kafka-versions

```diff
@@ -22,7 +22,7 @@
 # version default  proto   msg   sha512
 #1.0.2              1.0     1.0   4CBCDAF8CCC4EFE3D1B6275F3F2C32CF8F2F1A62104B5DD0BD9E2974160AB89D85A6E1791AF8B948A413B99ED696B06EA9D4299B27EA63C3F7318DABF5761144
 #1.1.1              1.1     1.1   2A1EB9A7C8C8337C424EEFED7BAAE26B3DACBA6A4AB8B64D9A7D5C6EE2CDB66CFA76C5B366F23435941569B89BF02482625189016296B2EA2A05FD0F38F6B709
-2.0.0              2.0     2.0   b28e81705e30528f1abb6766e22dfe9dae50b1e1e93330c880928ff7a08e6b38ee71cbfc96ec14369b2dfd24293938702cab422173c8e01955a9d1746ae43f98
+#2.0.0              2.0     2.0   b28e81705e30528f1abb6766e22dfe9dae50b1e1e93330c880928ff7a08e6b38ee71cbfc96ec14369b2dfd24293938702cab422173c8e01955a9d1746ae43f98
```

Comment:
Any particular reason for this? Or is it just a leftover from some test?

Reply:
I'll remove it, but the question is, in 0.12.x are we going to continue providing 2.0.0 when 0.11.x provides 2.0.1? There's little cost to having it, but it does slow down builds.

Reply:
I think we need to have some strategy for this which we will follow. We can't keep adding releases forever. 2.2.0 will be released soon. So I think we need to talk about things like when are we going to drop 2.0.X completely etc. Anyway, that is a bit beyond the scope of this PR I guess.